src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -34,6 +40,13 @@ public BookController(BookRepository bookRepository) {
      * @return {@link List<Book>}
      */
     @GetMapping
+    @ApiOperation(value = "return all book", response = Book[].class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success"),
+            @ApiResponse(code = 401, message = "Not Authorized"),
+            @ApiResponse(code = 403, message = "Access forbidden"),
+            @ApiResponse(code = 404, message = "Book Not Found"),
```

Comment:
This method can't return 404 because if there are no books, the return value will be an empty list with status 200-OK :)